### ISSUE TITLE
CSS data-URI support for older IE versions

### ISSUE THREAD

**Issue by andrewdavey** [andrewdavey/cassette#43]
IE 6 and 7 need some mhtml magic.

IE 8 - need to check for max image size and not convert those that are too big.

**Comment by thecontrarycat** [comment]
Don't know if you've done anything on this yet, Andrew, but I've made a start here (I haven't pushed to github yet as I have a pending pull request and don't want to mess it up).

So far I've changed the EmbedImages function to take an enum: {DataUri, IE8DataUri, Mhtml} so you can choose to have full embedding or not.  Mhtml rendering is next!

**Comment by andrewdavey** [comment]
I've not started any work on this yet, so I'd be very happy to see a pull request :)

**Comment by thecontrarycat** [comment]
Interesting dilemma: mhtml urls need to be fully qualified absolute paths:

`url(mhtml:http://www.phpied.com/files/datasprites/datasprites2.php!aol)`

This means we need to know at render time what the url of the css file will be.  Now, I can get the local part of the path from the UrlGenerator, but the `http://www.example.org` part will need to either be pulled from IIS somehow or entered as a configuration entry somewhere.  I'm in favour of adding a property to the `StylesheetBundle` that will be the fully qualified domain name used by the MHTML embedding.  

Any thoughts?

**Comment by andrewdavey** [comment]
Yeah, that's a potentially tricky issue. Go with whatever's simplest for now.

I imagine Cassette.Web.Startup will be able to get the domain name from the current HttpContext, but I may need to check this doesn't happen before HttpContext has been assigned.

If you can get your fork working with something basic for now, I can look at the fiddly integration stuff. Thank :)
